src/components/SignUpForm.jsx

```diff
@@ -0,0 +1,62 @@
+const SignUpForm = () => {
+  return (
+    <form action='' className='signup__form grid p-1'>
+      <div className='input__text-group mt-4'>
+        <input
+          type='email'
+          className='input__text-input'
```

Comment:
you can merge both signin and signup form in to one

Reply:
I didn't get that , how ?
Right now I am using two separate components for sign in and signup .Do you want me to do that from a single component ?